src/admin/context.ts

```diff
@@ -86,15 +86,10 @@ export class ContextManager implements IContext {
     );
     const hasReadOnly = accessGroups.includes(MozillaAccessGroup.READONLY);
 
-    // array to hold groups as strings because typescript is not happy with enums
-    const scheduledSurfaceAccessGroups = [
-      MozillaAccessGroup.NEW_TAB_CURATOR_ENUS as string,
-      MozillaAccessGroup.NEW_TAB_CURATOR_ENGB as string,
-      MozillaAccessGroup.NEW_TAB_CURATOR_DEDE as string,
-      MozillaAccessGroup.NEW_TAB_CURATOR_ENINTL as string,
-      MozillaAccessGroup.POCKET_HITS_CURATOR_ENUS as string,
-      MozillaAccessGroup.POCKET_HITS_CURATOR_DEDE as string,
-    ];
+    // converting access groups enum to an string array for typescript
+    const scheduledSurfaceAccessGroups = Object.values(MozillaAccessGroup).map(
+      (scheduledSurfaceGroup) => scheduledSurfaceGroup as string
+    );
```

Comment:
After a serious amount of debugging and rearranging all tests every which way, I reverted this change and all tests started passing again 🤷‍♀️

Reply:
This is so weird 😕 thanks for helping out @nina-py!